### PR TITLE
Use fake-phy-host as a the only host

### PR DESCRIPTION
**Pull request by jd**
This removes non existing host and make it work by default.

### CODE REVIEW

**Comment by sileht** [comment]
I have done this, to be able to do:

`docker-compose up --build --scale fake-phy-host=3 -d`

And got 3 hosts in the dashboard. But since the hostname is random, the example dashboard currently does not work :(

**Comment by jd** [comment]
@sileht all right… is there any other option? The current state is not user friendly, but I also don't want to make it hard for you to run this scenario.

File: grafana/dashboard.json [hunk 1]
```diff
@@ -374,8 +374,8 @@
                     "current": {
                         "selected": true,
                         "tags": [],
-                        "text": "fake-phy-host-5defe979e0eb",
-                        "value": "fake-phy-host-5defe979e0eb"
+                        "text": "fake-phy-host",
+                        "value": "fake-phy-host"
```
Comment (sileht) [inline grafana/dashboard.json]: Just put an empty dict, grafana will regenerate it from Gnocchi

File: grafana/dashboard.json [hunk 2]
```diff
@@ -385,19 +385,9 @@
                     "name": "server",
                     "options": [
                         {
```
Comment (sileht) [inline grafana/dashboard.json]: Same here with an empty list

File: collectd/run.sh [hunk 3]
```diff
@@ -2,7 +2,7 @@
 set -e
 exec >&2
 
-export COLLECTD_HOST=fake-phy-host-$(hostname -f)
```
Comment (sileht) [inline collectd/run.sh]: For a less random name:

```
NAME=$(sed -n "$((RANDOM%$(cat /usr/share/dict/words | wc -l))) p" /usr/share/dict/words)
export COLLECTD_HOST=fake-phy-host-$NAME
```

Comment (sileht) [inline collectd/run.sh]: Looks like words have more than 65000 words, we need a bigger RANDOM:

```
lines=$(cat /usr/share/dict/words | wc -l)
NAME=$(sed -n "$((RANDOM*$RANDOM%$lines)) p" /usr/share/dict/words)
export COLLECTD_HOST=fake-phy-host-$NAME
```

Comment (jd) [inline collectd/run.sh]: That solve no problem AFAIU. We can keep `hostname -f`, the real problem is the template that wants to "select" a host by default.

File: grafana/dashboard.json [hunk 4]
```diff
@@ -385,19 +385,9 @@
...
-                        },
-                        {
                             "selected": true,
-                            "text": "fake-phy-host-5defe979e0eb",
-                            "value": "fake-phy-host-5defe979e0eb"
+                            "text": "fake-phy-host",
+                            "value": "fake-phy-host"
                         }
                     ],
                     "query": "resources(collectd, host, host like \"fake-%\")",
```
Comment (sileht) [inline grafana/dashboard.json]: I got the root cause, just put `refresh:1` L404

**Comment by jd** [comment]
Pull-request updated, HEAD is now 737d32c450829ee02618e0953276502e4720113f

**Comment by sileht** [comment]
Do we need 2 reviewers for this repo ?

**Comment by jd** [comment]
Need is a strong word. I don't think @chungg approve so it's mostly me and you. I'd be ok to set it to just one.

**Pull request merged**